### PR TITLE
Bug 1848419 - Add custom chromium-as-release app for android

### PR DESCRIPTION
**Pull request by 92kns**
required for https://bugzilla.mozilla.org/show_bug.cgi?id=1848419, (android follow up of #7669)

### CODE REVIEW

File: schemas/performance-artifact.json [hunk 1]
```diff
@@ -14,9 +14,10 @@
             "refbrow",
             "fenix",
             "safari",
-            "custom-car"
+            "custom-car",
+            "custom-car-m"
           ],
-          "maxLength": 10,
+          "maxLength": 12,
```
Comment (gmierz) [inline schemas/performance-artifact.json:20]: This is limited in the database: https://github.com/mozilla/treeherder/blob/23a642030b126501e3a6000326a3fc9c0d384b8c/treeherder/perf/models.py#L57

Could we find a shorter name instead?

Comment (92kns) [reply schemas/performance-artifact.json:20]: ah I see. thanks @gmierz !
 how about `c-car-m`?

Comment (alexandru-io) [reply schemas/performance-artifact.json]: `cust-car-m`? It's 10 😄
`cst-cat-m` it's 9.
`c-cat-m` seems to me a bit confusing.

Comment (92kns) [reply schemas/performance-artifact.json]: cstm-car-m? 10 :)

**Review by alexandru-io — APPROVED** [review]

**Comment by 92kns** [comment]
thanks @alexandru-io !
@jmaher could you merge this please?

**Pull request merged**